src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -25,6 +30,14 @@
 
     private static final String FXML = "MainWindow.fxml";
 
+    private static final String QUICK_START_INSTRUCTIONS = "💡 Quick Tips:\n"
+            + AddCommand.USER_TIP + "\n\n"
+            + DeleteCommand.USER_TIP + "\n\n"
+            + LessonAddCommand.USER_TIP + "\n\n"
+            + ClearCommand.USER_TIP + "\n\n"
+            + HelpCommand.USER_TIP + "\n\n"
+            + "Have fun exploring TAB! ʕ•́ᴥ•̀ʔっ♡";
```

Comment:
![image](https://user-images.githubusercontent.com/35279431/136314641-6f3c467f-f99e-414e-b58d-b7b8a4d40eea.png)
It appears like this for me

Reply:
similarly, "exploring" seems ok for new users but a little strange for long time users.
How about "Have fun using TAB!"